src/backend/app/config.py

```diff
@@ -275,7 +275,38 @@ def set_raw_data_api_auth_none(cls, v: Optional[str]) -> Optional[str]:
             return None
         return v
 
-    # Used for temporary auth feature
+    ALGORITHM: str = "RS256"
+    AUTH_PRIVATE_KEY_PATH: str
+    AUTH_PUBLIC_KEY_PATH: str
+    AUTH_PRIVATE_KEY: Optional[str] = None
+    AUTH_PUBLIC_KEY: Optional[str] = None
+
+    @field_validator("AUTH_PRIVATE_KEY", mode="before")
+    @classmethod
+    def load_private_key(cls, v: Optional[str], info: ValidationInfo) -> str:
+        """Loads the private key for authentication."""
+        if v:
```

Comment:
Shouldn't this be `if v is None` or am I missing something?

We only load from the file if the variable is not set directly.

Reply:
Yes, you are actually right.